cylc/uiserver/authorise.py

```diff
@@ -225,7 +225,7 @@ def get_access_user_permissions_from_owner_conf(
         allowed_operations.discard("")
         return allowed_operations
 
-    @lru_cache(maxsize=128)
+    @lru_cache(maxsize=128)  # noqa B019 TODO?
```

Comment:
TODO: this cache can keep instances of the `Authorization` class alive after the instances would otherwise have been garbage collected.

Since only one `Authorisation` instance is created per UIS and since it lives for the lifetime of the UIS this is not a problem, however, we may as well do it properly.

Reply:
https://github.com/cylc/cylc-uiserver/issues/330